tardis/apps/publication_workflow/static/publication-actions/publication_actions.js

```diff
@@ -0,0 +1,269 @@
+     *
+     */
+    vm.createPublication = function () {
+        vm.openPublicationForm();
+    };
+
+    /**
+     * Opens the publication form modal dialog to resume editing a publication
+     *
+     */
+    vm.resumePublication = function () {
+        $log.debug("resumePublication: experimentId: " + vm.experimentId);
+        var formData = vm.emptyFormData();
+        formData.publicationId = vm.experimentId;
+        $log.debug("resumePublication: formData.publicationId: " + formData.publicationId);
+        formData.action = "resume";
+        $http.post('/apps/publication-workflow/form/', formData).then(function (response) {
+            $log.debug("resumePublication.http.post.success: response.data.publicationId: " + response.data.publicationId);
+            $log.debug("resumePublication.http.post.success: response.data.publicationTitle: " + response.data.publicationTitle);
+            vm.formData = response.data;
+            if (angular.isString(vm.formData.embargo)) {
+                vm.formData.embargo = new Date(vm.formData.embargo);
+            }
+            vm.openPublicationForm();
+        },
+        function () {
+            vm.errorMessages = ['Could not load publication draft!'];
+        });
+    };
+
+    /**
+     * Open tokens dialog, allowing creation of temporary links for sharing.
+     *
+     * The publicationId (a.k.a. experimentId) is passed to the view URL
+     * so the view method can set is_owner in the context passed to the
+     * tokens.html Django template.  The publicationId is also passed via the
+     * ngDialog's data to the PublicationTokensController instance.
+     */
+    vm.sharePublication = function () {
+        ngDialog.open({
+            template: '/apps/publication-workflow/tokens/' + vm.experimentId + '/',
+            data: {
+                'publicationId': parseInt(vm.experimentId)
+            },
+            closeByDocument: false,
+        });
+    };
+
+    /*
+     * Delete publication draft
+     */
+    vm.deletePublicationDraft = function () {
+        var confirmation = $mdDialog.confirm()
+            .title('Are you sure you want to delete Publication ID ' + vm.experimentId + '?')
+            .textContent('You cannot undo this action!')
+            .ok('Yes, delete it!')
+            .cancel('No, keep it.');
+  
+        $mdDialog.show(confirmation).then(function() {
+            $log.info('OK, deleting publication...');
+            $http.post('/apps/publication-workflow/publication/delete/' + vm.experimentId + '/', {})
+               .then(function (_response) {
+                   $log.debug("Publication deleted successfully.");
+                   $window.location.reload();
+               },
+               function(_response) {
+                   $log.debug("Failed to delete publication.");
+               });
+        }, function() {
+            $log.info('OK, keeping publication');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933951)
```suggestion
            $log.info("OK, keeping publication");
```